### DISCUSSION TITLE
Command "CI='' npm run build" exited with 126

### DISCUSSION THREAD

**Discussion by furkansahinz** [vercel/community#70] [Help] [answered]
19:10:17.315 | Cloning completed: 4.140s
19:10:20.550 | Analyzing source code...
19:10:22.584 | Installing build runtime...
19:10:25.798 | Build runtime installed: 3.214s
19:10:27.893 | Looking up build cache...
19:10:28.175 | Build Cache not found
19:10:28.932 | Detected package.json
19:10:28.932 | Installing dependencies...
19:11:03.727 |  
19:11:03.727 | > core-js@2.6.12 postinstall /vercel/path0/front/node_modules/babel-runtime/node_modules/core-js
19:11:03.728 | > node -e "try{require('./postinstall')}catch(e){}"
19:11:03.728 |  
19:11:03.827 |  
19:11:03.828 | > core-js@3.18.1 postinstall /vercel/path0/front/node_modules/core-js
19:11:03.828 | > node -e "try{require('./postinstall')}catch(e){}"
19:11:03.828 |  
19:11:03.900 |  
19:11:03.901 | > core-js-pure@3.18.1 postinstall /vercel/path0/front/node_modules/core-js-pure
19:11:03.901 | > node -e "try{require('./postinstall')}catch(e){}"
19:11:03.901 |  
19:11:03.993 |  
19:11:03.993 | > node ./postinstall.js
19:11:03.993 |  
19:11:05.873 | npm WARN @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.15.4 requires a peer of @babel/core@^7.13.0 but none is installed. You must install peer dependencies yourself.
19:11:05.880 | npm WARN frontend@0.1.0 No description
19:11:05.885 | npm WARN frontend@0.1.0 No repository field.
19:11:05.891 | npm WARN frontend@0.1.0 No license field.
19:11:05.900 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/webpack-dev-server/node_modules/fsevents):
19:11:05.900 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:11:05.910 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
19:11:05.910 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:11:05.920 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
19:11:05.921 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:11:05.921 |  
19:11:05.926 | added 1923 packages from 873 contributors in 36.539s
19:11:07.087 |  
19:11:07.087 | 167 packages are looking for funding
19:11:07.087 | run `npm fund` for details
19:11:07.087 |  
19:11:07.529 |  
19:11:07.529 | > frontend@0.1.0 build /vercel/path0/front
19:11:07.529 | > craco build
19:11:07.529 |  
19:11:07.534 | sh: /vercel/path0/front/node_modules/.bin/craco: Permission denied
19:11:07.538 | npm ERR! code ELIFECYCLE
19:11:07.538 | npm ERR! errno 126
19:11:07.543 | npm ERR! frontend@0.1.0 build: `craco build`
19:11:07.543 | npm ERR! Exit status 126
19:11:07.543 | npm ERR!
19:11:07.543 | npm ERR! Failed at the frontend@0.1.0 build script.
19:11:07.544 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
19:11:07.548 |  
19:11:07.548 | npm ERR! A complete log of this run can be found in:
19:11:07.548 | npm ERR!     /vercel/.npm/_logs/2021-12-11T16_11_07_544Z-debug.log
19:11:07.554 | Error: Command "CI='' npm run build" exited with 126

**Comment by MaEdAlpha** [comment]
I am also having this issue.

**Comment by Pieparker** [comment, marked as answer]
Hey there! I'd recommend that you check out [this discussion](https://github.com/gsoft-inc/craco/issues/308) in the `craco` repo for help with `craco`. TL:DR; you may want to remove your `node_modules` folder; perhaps something is cached that oughtn't be.

**Reply by lordW1lly** [reply]
I had the same issue, I removed node_modules and build.

**Reply by timenyin** [reply]
thanks it worked
